style/Makefile

```
@@ -20,5 +21,16 @@ config.js:
 code_format:
 	npx prettier --write .
 
-run: sprites config.js
+build/maplibre-js:
+	mkdir -p build/maplibre-js
+	git clone git@github.com:wipfli/maplibre-gl-js.git build/maplibre-js
```

This unnecessarily requires having an SSH key on file with github, which shouldn't be needed for an anonymous clone.
```suggestion
	git clone https://github.com/wipfli/maplibre-gl-js build/maplibre-js
```